### PR TITLE
SQL update for non-standard codes PC and DPC.

### PR DESCRIPTION
**Pull request by shanmukhasampath**
Updated the 5 SQL files that are necessary to be able to get non-standard codes Person Count (PC) and DPC.

Tested it out on development side and it works.

### CODE REVIEW

File: src/main/resources/ddl/achilles/achilles_result_concept_count.sql [hunk 1]
```diff
@@ -83,36 +72,56 @@ WHERE analysis_id IN (405, 605, 705, 805, 807, 1805, 1807, 2105)
...
+    UNION
+    -- include any ancestor concept that has a descendant in counts
+    select distinct cast(ancestor_concept_id as varchar(50)) concept_id
+    from counts c
+    join @vocab_schema.concept_ancestor ca on cast(ca.descendant_concept_id as varchar(50)) = c.concept_id
+  ) c
+  left join @vocab_schema.concept_ancestor ca on c.concept_id = cast(ca.ancestor_concept_id as varchar(50))
+)
 INSERT INTO @results_schema.achilles_result_concept_count (concept_id, record_count, descendant_record_count, person_count, descendant_person_count)
 SELECT
```
Comment (RigbeGW) [inline src/main/resources/ddl/achilles/achilles_result_concept_count.sql]: I just compared these changes with the latest version of the webapi and here they select DISTINCT, does it matter in our case?

**Review by RigbeGW — COMMENTED** [review]
All the changes to the other files are on par with the latest webapi code. I made a comment on the achilles_result file that we are not selecting the DISTINCT tuples and if that doesnt matter in our case, we can merge this PR.

File: src/main/resources/ddl/achilles/achilles_result_concept_count.sql (hunk 1 above)
Comment (shanmukhasampath) [reply src/main/resources/ddl/achilles/achilles_result_concept_count.sql]: Hi Rigbe, thank you for checking. The DISTINCT does not make a difference but still I will add "DISTINCT" and push the change.

**Review by RigbeGW — APPROVED** [review]
looks good

**Pull request merged**